schema/generator.go

```diff
@@ -787,6 +787,9 @@ func (g *Generator) generateAddIndex(table string, index Index) string {
 		if indexColumn.length != nil {
 			column += fmt.Sprintf("(%d)", *indexColumn.length)
 		}
+		if indexColumn.direction == "desc" {
```

Comment:
This string comparison seems fragile (e.g. against "DESC"). Can we use bool or some more restrictive type than this?

Reply:
I changed the approach to use constants.
`asc_desc_opt` in the syntax file also returns the same constant, so the comparison here is now robust.